app/models/assignment_participant.rb

```diff
@@ -1,6 +1,6 @@
 require 'uri'
 require 'yaml'
-
+require 'TFD1770_refactor'
```

Comment:
According to [this configuration](https://github.com/expertiza/expertiza/blob/master/config/application.rb#L29), maybe you do not need to require lib file.

Reply:
I found that I have to add "require". If I delete "require", the test cannot be passed. 
![image](https://user-images.githubusercontent.com/31051241/32310915-20eb8356-bf6c-11e7-88bc-c294f810f1c0.png)